cypress/integration/instruments.ts

```diff
@@ -1,9 +1,167 @@
 context('Instrument tests', () => {
   const faker = require('faker');
 
+  function createInstrument({
```

Comment:
Isn't it better to add these functions as commands or they are used just in this file? I guess we use createInstrument and createCall in other files as well.

Reply:
In long term that would be the goal, but for now I didn't bother with it. Maybe in a dedicated task.